src/main/java/org/folio/event/EdiExportHistoryEventType.java

```diff
@@ -5,8 +5,9 @@
 
 @RequiredArgsConstructor
 @Getter
-public enum ExportEventType {
-  EXPORT_HISTORY_CREATE("edi-export-history.create");
+public enum EdiExportHistoryEventType {
+  EXPORT_HISTORY_CREATE("edi-export-history.create"),
+  INVENTORY_ITEM_CREATE("inventory.item");
```

Comment:
INVENTORY_ITEM_CREATE should not be in EdiExportHistoryEventType

Reply:
Thanks, removed